app/human-detector-app/src/helpers.ts

```diff
@@ -0,0 +1,15 @@
+import * as ReactNative from 'react-native';
+import * as Device from 'expo-device';
+
+/**
+ * This file will help make testing easier, and organize some helpers
+ * for any expo or react native commands.
+ */
```

Comment:
It was a recommendation to do this instead of mocking the dependencies since I was having weird problems with mocking some of these variables.